lib/ui/painting.dart

```diff
@@ -1519,6 +1519,13 @@ class Path extends NativeFieldWrapperClass2 {
   }
   void _addOval(double left, double top, double right, double bottom) native 'Path_addOval';
 
+  /// Adds a new subpath that consists of a curve that forms the
+  /// circle centered on centerX,centerY with specified radius
```

Comment:
This should end with a period. Also, put backticks around references to arguments.